diff --git a/teams-extension.php b/teams-extension.php
index 451f2fb..070c452 100644
--- a/teams-extension.php
+++ b/teams-extension.php
@@ -43,7 +43,7 @@ class Auth_OpenID_TeamsExtension extends Auth_OpenID_Extension {
    * Get the string arguments that should be added to an OpenID
    * message for this extension.
    */
-  function getExtensionArgs() {
+  function getExtensionArgs($request = null) {
     $args = array();
 
     if ($this->_teams) {
@@ -58,7 +58,7 @@ class Auth_OpenID_TeamsExtension extends Auth_OpenID_Extension {
    *
    * Returns the message with the extension arguments added.
    */
-  function toMessage(&$message) {
+  function toMessage($message, $request = null) {
     if ($message->namespaces->addAlias($this->ns_uri, $this->ns_alias) === null) {
       if ($message->namespaces->getAlias($this->ns_uri) != $this->ns_alias) {
         return null;
